setup.py

```diff
@@ -47,7 +51,10 @@
     ],
     packages=find_packages(),
     python_requires=">=3.7",
-    extras_require={"docs": ["sphinx", "sphinx-bluebrain-theme"]},
+    extras_require={
+        "docs": ["sphinx", "sphinx-bluebrain-theme"],
+        "krb": EXTRA_KRB,
```

Comment:
I think maybe having "alt" for the keyrings.alt would be nice. Although, I'd rather have something like "bb5" (no matter what it is called, really), that would install all the needed for BB5.

Reply:
yeah, something like that could work.  I'm still unclear on how `keyring` works on macos